Formula/c/c_formatter_42.rb

```diff
@@ -0,0 +1,32 @@
+class CFormatter42 < Formula
+  include Language::Python::Virtualenv
+
+  desc "C language formatter for 42 norminette"
+  homepage "https://github.com/dawnbeen/c_formatter_42"
+  url "https://files.pythonhosted.org/packages/77/fa/edee608023d6adc567d1060b4b889b6279e0f99ff3b3cc58bfa4eda069dd/c_formatter_42-0.2.8.tar.gz"
+  sha256 "da31b1ab4cdc92dd363ae791244cf063982e29c9fc54dbde0f22fce20a2b057b"
+  license "GPL-3.0-only"
+  head "https://github.com/dawnbeen/c_formatter_42.git", branch: "master"
+
+  depends_on "python"
+
+  def install
+    python_version = Language::Python.major_minor_version "python3"
+    odie "Python 3.8 or newer is required." if python_version < Version.new("3.8")
```

Comment:
We don't want to odie in the install, especially because the current python is https://formulae.brew.sh/formula/python@3.13#default
```suggestion
```